tests/tests_geomstats/test_invariant_metric.py

```diff
@@ -30,6 +29,7 @@ class TestInvariantMetric(RiemannianMetricTestCase, metaclass=Parametrizer):
     skip_test_log_then_exp = np_backend()
     skip_test_geodesic_bvp_belongs = True
     skip_test_exp_then_log = True
+    skip_test_dist_point_to_itself_is_zero = True
```

Comment:
This does not work? :/ 
what is the error and did you raise an issue on it?

Reply:
yeah.. sure I will raise it